cli/dcoscli/data/help/package.txt

```diff
@@ -0,0 +1,48 @@
+Install and manage DCOS package versions
```

Comment:
why did this change?  they're not managing versions.  they're managing packages.